### ISSUE TITLE
Stats overview breaks eventually

### ISSUE THREAD

**Issue by thrize** [Steem-FOSSbot/steem-fossbot-voter#63]
If it has run more than ```DAYS_KEEP_LOGS``` days, the redis DB _cannot_ delete old keys due to incorrect redis library ```del``` command usage.

**Comment by thrize** [comment]
Fixed in v0.2.8

**Comment by thrize** [comment]
This seems to be broken again, investigate and fix

**Comment by thrize** [comment]
Fixed in v0.2.8

**Comment by thrize** [comment]
Was breaking for dashboard (web app) initiated runs

**Comment by thrize** [comment]
Fixed in v0.2.8

**Comment by thrize** [comment]
Seems fixed now, as of v0.2.8

Note this will become obsolete when #64 (switch to MongoDB) is completed

**Comment by thrize** [comment]
Nope, this is not fixed. The record management is not working, old data is not getting deleted.

```
app[heroku-redis]: source=REDIS_URL sample#used_memory_over_limit=1728432.0bytes message=Database memory close to the limit. Please upgrade your database plan or change your maxmemory policy.
app[heroku-redis]: source=REDIS sample#active-connections=1 sample#load-avg-1m=0.045 sample#load-avg-5m=0.09 sample#load-avg-15m=0.11 sample#read-iops=0 sample#write-iops=0 sample#memory-total=15664292kB sample#memory-free=12655728kB sample#memory-cached=750548kB sample#memory-redis=25321392bytes sample#hit-rate=0.90688 sample#evicted-keys=0
```

Fix

**Comment by thrize** [comment]
Fix applied, need to review

**Comment by thrize** [comment]
Fix not sufficient, check again

**Comment by thrize** [comment]
Fix in which should take care of orphan metadata in DB

**Comment by thrize** [comment]
Confirmed fixed in v0.2.8
